python_docs_theme/static/menu.js

```diff
@@ -0,0 +1,22 @@
+
+document.addEventListener('DOMContentLoaded', function () {
+    const toggler = document.querySelector('.toggler');
+    const sideMenu = document.querySelector('.menu-wrapper');
+    const doc = document.querySelector('.document');
+    function closeMenu() {
+        sideMenu.classList.remove('open');
+        toggler.checked = false;
+    }
+    toggler.addEventListener('change', function (e) {
+        if (toggler.checked) {
+            sideMenu.classList.add('open');
+        } else {
+            closeMenu();
+        }
+    });
+    doc.addEventListener('click', function () {
+        if (toggler.checked) {
+            closeMenu();
+        }
+    })
+})
```

Comment:
By moving the checkbox up in the dom tree it may be possible to implement this using only javascript and make it more accessible for people without javascript

Reply:
I replaced the checkbox with a button. Is this better for people without javascript?

Reply:
What I would have done is move the input as a checkbox directly next to the sidebar, reference it in the label via id and apply the styles based on whether the checkbox is checked or not. I may have time tomorrow to create a PR on your fork repo if you want to.

Reply:
I have done it like this at first (checkbox input and applying styles based on :checked property). Then I was checking for accessibility and found [this article](http://www.ashleysheridan.co.uk/blog/Making+an+Accessible+Hamburger+Menu) which shows that a button as menu opener has some advantages regarding accessibility: namely, it is easier to set up keyboard handling. On the other hand, I just realized that people probably don't use keyboard for navigation on mobile, and it is more important to set up No-script solution than keyboard navigation on mobile. 
Also, I looked at other documentation sites: readthedocs theme and VueJS docs, and their hamburger menu doesn't work without javascript.

Reply:
In addition if you follow [this guide](https://www.silvestar.codes/articles/css-sidebar-toggle/) you also get a working css only solution with added accesibility using javascript